sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryBaseTable.scala

```diff
@@ -273,7 +275,24 @@ abstract class InMemoryBaseTable(
     }
   }
 
-  case class InMemoryStats(sizeInBytes: OptionalLong, numRows: OptionalLong) extends Statistics
+  case class InMemoryStats(
+      sizeInBytes: OptionalLong,
+      numRows: OptionalLong,
+      override val columnStats: Optional[HashMap[NamedReference, ColumnStatistics]])
+    extends Statistics
+
+  case class InMemoryColumnStats (
+      override val distinctCount: Optional[BigInteger],
```

Comment:
Indentation?

Reply:
Should this be 4-space indentation? 
I have an extra space after `InMemoryColumnStats`, will remove.